qaframework-bdd-tests/src/test/resources/features/refapp-2.x/016-XssVulnerability/xssVulnerability.feature

```diff
@@ -0,0 +1,24 @@
+Feature: Check Xss Vulnerability
+
+  Background:
+    Given User goes to active visit search page
+   
+  @selenium
+  @xssVulnerability
+  Scenario: Check XSS Vulnerability
+    When User searches for patient using patient identifier
+    And User goes to patient dashboard of last Active visit
+    Then User clicks on show contact link
+    And User clicks on editContact info
+    And User clicks on phone Number edit
+    Then User clears phone number
+    And User enters new phone number with xss check script
+    And User clicks on confirm Edit button
+    Then System returns a message validator
+
```

Comment:
any reason for adding an empty line here?

Reply:
The slight reason would be to clearly follow the scenarios properly since they are gona be compact.Another reason is that since the test clears the input text twice, i would love seeing where it clears again, so  i add another line for formality and clean code presumably.

Reply:
does it each step tells clearly what is being done?

Reply:
Resolved thanks. Though it doesnt look morethan good . Remember these are human readable scenarios. i personally discourage passages without paragraphs.

Reply:
why do you think there should be an empty line? And when should an empty line be added within the feature file?